extension/src/experiments/workspace.ts

```diff
@@ -329,9 +329,19 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
   }
 
   public getHasData() {
-    return Object.values(this.repositories).some(repository =>
-      repository.getHasData()
-    )
+    const allLoading = undefined
+
+    const repositories = Object.values(this.repositories)
+
+    if (repositories.some(repository => repository.getHasData())) {
+      return true
+    }
+
+    if (repositories.some(repository => repository.getHasData() === false)) {
+      return false
+    }
+
+    return allLoading
```

Comment:
[F] Needed for the edge case that the current project has data but CLI needs to be installed. I.e [this](https://github.com/iterative/vscode-dvc/pull/2944/files#diff-f1285e681b370f15ccca1f8455d2009a0dcde397464d0e5092282a932ad4962cR78)